src/tests/class_tests/openms/source/DIAScoring_test.cpp

```diff
@@ -439,32 +452,32 @@ START_SECTION (void dia_massdiff_score(const std::vector< TransitionType > &tran
   normalized_library_intensity.push_back(0.7);
   normalized_library_intensity.push_back(0.3);
   std::vector<double> ppm_errors;
-  diascoring.dia_massdiff_score(transitions, sptr, normalized_library_intensity, ppm_score, ppm_score_weighted, ppm_errors);
+  diascoring.dia_massdiff_score(transitions, sptr, normalized_library_intensity, ppm_score, ppm_score_weighted, ppm_errors, -1, -1);
 
   TEST_REAL_SIMILAR(ppm_score, (15 + 10) / 2.0); // 15 ppm and 10 ppm
   TEST_REAL_SIMILAR(ppm_score_weighted, 15 * 0.7 + 10* 0.3); // weighted
 }
 END_SECTION
 
-START_SECTION ( bool DIAScoring::dia_ms1_massdiff_score(double precursor_mz, transitions, SpectrumType spectrum, double& ppm_score) )
+START_SECTION ( bool DIAScoring::dia_ms1_massdiff_score(double precursor_mz, transitions, std::vector<SpectrumType> spectrum, double& ppm_score, double drift_start, double drift_end) )
```

Comment:
the new arguments are never tested?!
Somewhere else maybe?
Add a comment where?